src/pages/FriendsActivitiesPage.js

```diff
@@ -58,3 +90,20 @@ export default function FriendsActivitiesPage({
     }
   }
 }
+const ScrollCategories = styled.span`
```

Comment:
I don't think that a `span` is semantically the correct element here.

Reply:
Thank you, I completely forgot about proper HTML semantic🙈